### ISSUE TITLE
Type '{ children: Element; }' has no properties in common with type 'IntrinsicAttributes & { url?: Url | undefined; async?: boolean | undefined; } & MemoryRouterEventHandlers'.

### ISSUE THREAD

**Issue by hcharley** [scottrippey/next-router-mock#40]
# Usage

```tsx
import { MemoryRouterProvider } from 'next-router-mock/MemoryRouterProvider';

...
    <MemoryRouterProvider>
       <></>
    </MemoryRouterProvider>
...
```

# Conjecture 

I take it this is related to React 18's dropping of the "implicit" `children` prop from FC:

https://solverfox.dev/writing/no-implicit-children/

**Comment by scottrippey** [comment]
Published as version `0.6.6`
Please report if this fixes the issue.  Thanks!

**Comment by hcharley** [comment]
Fixed! 🎊
